wiki/Tournaments/Countries_that_participated_in_osu!_tournaments/en.md

```diff
@@ -1,359 +1,362 @@
----
-outdated: true
----
-<!-- TODO: outdated because the numbers haven't accounted for any recent tournaments. not sure if this article should exist anymore... -->
-
 # Countries that participated in osu! tournaments
 
-This article lists the countries that participated in the various osu!, osu!taiko, CtB and osu!mania tournaments as well as their rank if they finished in the top 4.
-
-**Please note:** some tournaments are not listed here because either they were allowing more than 2 countries in a single team or teams were not representing countries.
+This article lists countries that have participated in official osu!, osu!taiko, osu!catch, and osu!mania tournaments. It also provides the rankings of teams that have reached the top 3.
 
-**Marks used in the Attendance sections**
+*Note: Official tournaments which were allowing teams not representing countries are not taken into consideration.*
 
-| Mark | Explantation |
-| :-: | :-- |
-| ![][true] | Participated |
-| ![][false] | Didn't participate |
+*Note: Mixed-country teams (e.g. Argentina-Columbia) are not taken into consideration.*
```

Comment:
kinda feel like these two notes can be merged together, they both seem somewhat related

Reply:
it would leave them seperated as i believe that a combined note would sound confusing as they relate to somehow different things